.github/workflows/deploy.yml

```diff
@@ -19,15 +19,21 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: "18.20.2"
+          node-version: "22.8.0"
```

Comment:
is this the node version that you are using? Make sure they are the same to avoid any surprise issues.

Reply:
Yes it is. I ended updated node between when I first made the deploy file and now.